ath-container.sh

```diff
@@ -6,8 +6,9 @@ trap 's=$?; echo "$0: Error $s on line "$LINENO": $BASH_COMMAND"; exit $s' ERR
 uid=$(id -u)
 gid=$(id -g)
 tag="jenkins/ath"
+java_version="${java_version:-8}"
 
-docker build --build-arg=uid="$uid" --build-arg=gid="$gid" src/main/resources/ath-container -t "$tag"
+docker build --build-arg=uid="$uid" --build-arg=gid="$gid" --build-arg=java_version="$java_version" src/main/resources/ath-container -t "$tag"
```

Comment:
In PCT there was a decision to bubdle both Java versions into a single container: https://github.com/jenkinsci/plugin-compat-tester/pull/86 . It helps to ship an official PCT image which can be configured to use different versions depending on options. It would also allow running ATH engine in Java 8 while using Java 8 only for Jenkins itself

CC @raul-arabaolaza who was experimenting with ATH images iirc

Reply:
Yeah, we would have to mimic that as I doubt we can reuse that container. I prefer to have ATH running on Java 11 and for now I am making progress so this is a good alternative in case I get stuck.

Reply:
I suspect that Selenium upgrade by @lvotypko may become a prerequisite. But it is merged in https://github.com/jenkinsci/acceptance-test-harness/pull/457

Reply:
@oleg-nenashev, exaclty

Reply:
@oleg-nenashev @olivergondza FTR I am strongly against moving the `latest` tag of `jenkins/ath` docker image to java 11. That may have a strong impact on third parties and break a lot of builds as not everybody may be using latest version of the ATH sources ready for java 11 but they may be using latest version of the docker image. At least I would ask for keeping both versions by building and publishing a `java8` version of the image or including both JDKs in the image and let users decide which one to use like in the PCT example as suggested by Oleg